### ISSUE TITLE
Realm with Kotlin 1.8.0(KotlinFrontEndException)

### ISSUE THREAD

**Issue by SomeBodyFromSimf** [realm/realm-kotlin#1227]
After update to kotlin 1.8.0 i have a problem
Full logs in attach
[Logs.txt](https://github.com/realm/realm-kotlin/files/10465122/Logs.txt)

**Comment by edualonso** [comment]
Hi @SomeBodyFromSimf. Could you please fill out the issue form? It will help us understand more about the problem you are experiencing.

**Comment by edualonso** [comment]
Oh sorry, I thought we had a template form you could fill out. Can you provide more information about your setup? What version of the Kotlin SDK you are using, version of Android Studio, whether you are using it in multiplatform mode or only for Android, Gradle plugin, etc.

**Comment by rorbech** [comment]
It seems that it fails with 
```
Caused by: java.lang.ClassNotFoundException: org.jetbrains.kotlin.resolve.extensions.SyntheticResolveExtension$DefaultImpls
```
Which should be fixed by #1159. Which version of realm are you using? If not using the latest could you try to upgrade to 1.5.1 or newer.

**Comment by SomeBodyFromSimf** [comment]
@rorbech thank you. It s works
